firmware/ec/src/devices/uart/UartMon.c

```diff
@@ -41,11 +41,18 @@ static UART_Handle UartMon_open(UART_Handle handle, UART_Params *params)
         return NULL;
     }
 
+#if 0
     /* This is a driver limitation - we only support blocking mode for now */
     if (params->readDataMode != UART_MODE_BLOCKING ||
         params->writeDataMode != UART_MODE_BLOCKING) {
         return NULL;
     }
+#else
```

Comment:
Why this change?

Reply:
We get a warning because readDataMode and UART_MODE_BLOCKING are different types.  Both are enums with values of 0 and 1.  Logically, this is the same as just testing if either readDataMode or writeDataMode is true.  If the change is OK, I'd rather remove the #if 0 part.  Thoughts?

Reply:
Left code as was and used pragma instead.  Warning is now gone.